skopt/dummy_search.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+from sklearn.utils import check_random_state
+
+from scipy.optimize import OptimizeResult
+
+def dummy_search(func, bounds=None, maxiter=1000, random_state=None):
```

Comment:
Can we try to enforce some consistency on naming? What do you think of `dummy_minimize` instead?